python/tank/util/shotgun.py

```diff
@@ -398,7 +398,7 @@ def create_sg_app_store_connection():
     return g_app_store_connection
 
 
-def __get_app_store_connection_information():
+def _get_app_store_connection_information():
```

Comment:
Ah. Silly me. When I was testing I got frustrated with decorating the name trying to call the function  so I removed an underscore. I would have called the calling method, which is public, but it would have been mocking even more stuff, but I've you've seen the unit tests, mocking more stuff is not a direction we want to get into for those tests. There's already enough.

Shall I leave it this way?